lib/modules/auth/login/bloc/login_state.dart

```diff
@@ -0,0 +1,36 @@
+import '/models/login/login_response_model.dart';
+
```

Comment:
I told you to make SucessState, FauilerState, ErrorState, and NetworkErrorState.
so you have a reason not to make these states

Reply:
can't understand when to return each state
I have to check the type of error and return accordingly?